kernel/data-pipeline/cdc/core/src/main/java/org/apache/shardingsphere/data/pipeline/cdc/api/impl/CDCJobAPI.java

```diff
@@ -286,7 +288,12 @@ protected YamlPipelineJobConfiguration swapToYamlJobConfiguration(final Pipeline
     
     @Override
     protected PipelineJobInfo getJobInfo(final String jobId) {
-        throw new UnsupportedOperationException();
+        JobConfigurationPOJO jobConfigPOJO = getElasticJobConfigPOJO(jobId);
+        PipelineJobMetaData jobMetaData = new PipelineJobMetaData(jobId, !jobConfigPOJO.isDisabled(),
+                jobConfigPOJO.getShardingTotalCount(), jobConfigPOJO.getProps().getProperty("create_time"), jobConfigPOJO.getProps().getProperty("stop_time"), jobConfigPOJO.getJobParameter());
+        CDCJobConfiguration jobConfig = getJobConfiguration(jobConfigPOJO);
+        List<String> tableNames = jobConfig.getSchemaTableNames().stream().map(each -> String.join(".", jobConfig.getDatabase(), each)).collect(Collectors.toList());
```

Comment:
logic database name could be put in another column, since there's only one database name per job, there might be too much duplicated text when there're much tables.